src/components/authentication/authentication.resolver.ts

```diff
@@ -33,8 +36,15 @@ export class AuthenticationResolver {
       defaultValue: false,
     })
     browser?: boolean
-  ): Promise<CreateSessionOutput> {
-    const token = await this.authService.createToken();
+  ): Promise<SessionOutput> {
+    // TODO: the 'browser' ternary check here isn't necessary but the e2e tests fail without it due to the cookieParser middleware not being loaded in the e2e app module
+    const existingToken =
+      this.sessionPipe.getTokenFromAuthHeader(req) ||
+      (browser ? this.sessionPipe.getTokenFromCookie(req) : null);
```

Comment:
Is it just `cookies` not existing on request? Could you just change the impl to this?
```ts
  getTokenFromCookie(req: Request): string | undefined {
    return req.cookies?.[this.config.session.cookieName];
  }
```

Reply:
good call! this will work